conditional/blueprints/attendance.py

```diff
@@ -272,6 +276,10 @@ def submit_house_attendance(user_dict=None):
                 meeting.id,
                 None,
                 m['status']))
+            if m['status'] == 'Absent':
+                absent_members.append(m['uid'])
+        thread = Thread(target=send_absent_hm_attendance_emails, name="HM Emails", args=(absent_members,))
+        thread.start()
```

Comment:
If you aren't going to reference thread again, then there's no need to assign it, just call start on the created thread.

Reply:
tbh I think this is more readable than ```Thread(target=send_absent_hm_attendance_emails, name="HM Emails", args=(absent_members,)).start()```